src/Centreon/Domain/Menu/Model/Page.php

```diff
@@ -107,4 +109,23 @@ public function isReact(): bool
     {
         return $this->isReact;
     }
+
+    /**
+     * Return the redirection uri of the page.
+     *
+     * @return string
+     */
+    public function getRedirectionUri(): string
+    {
+        if ($this->isReact) {
+            return $this->url;
+        }
+
+        $redirectionUri = self::LEGACY_PAGE_BASE_URI . (string) $this->pageNumber;
```

Comment:
I'm not sure, but I think it is not mandatory to cast pageNumber
If it is something noticed by phpstan, you can resolve my comment :)

Reply:
indeed

Reply:
i've removed the cast as phpstan don't notice anything in level max